client/src/components/Navbar/Navbar.css

```diff
@@ -1,4 +1,133 @@
+a:hover, a:visited, a:link, a:active{
+   text-decoration: none;
+   color: inherit;
+}
+
+.bg-nav{
+      opacity: 0.8;
+}
+
 .navbar-component .active {
-    font-weight: 900;
-    color: black;
-}
\ No newline at end of file
+      font-size: 1.2rem;
+      font-weight: 900;
+      color: white;
+}
+.navbar{
+      background-color: #080808;
+      padding: 3px 15px 3px 10px;
+}
+#navLink{
+      font-weight: 400;
+      font-style: italic;
+      color: #fff;
+}
+
+
+.nav-item{
+     color:#fff;
+     margin: 10px 25px;
+}
+
+.nav-logo{
+      opacity: 1;
+      height: 45px;
+      margin-left: 40px;
+}
+
+.navbar-toggler{
+      background-color: #7e8a97;
+}
+
+.search-bar{
+      border-radius:35px;
+      width:180px!important;
+      height: 30px;
+      margin-right: 25px;
+      margin-left: 20px;
+      padding-left: 35px;
+      }
+
+.search-icon{
+
+      position:absolute;
+      opacity: 0.6;
+      right: 190px;
+      bottom: 19px;
+      height: 19px;
+      }
+
+@media  (max-width:600px){
+
+.nav-logo{
+      margin-left: 10px;
+}
+
+
+}
+
+@media only screen and (max-width:1024px){
+      .search-bar{
+            display:none!important;
+      }
+
+      .search-icon{
+            display: none;
+      }
+}
+
+@media only screen and (min-width: 1500px){
+      .navbar{
+            height: 80px;
+}
+      .navbar-component{
+            font-weight: 900;
+            font-size: 1.4rem;
+            color: white;
+      }
+      .navbar-component .active {
+            font-size: 1.7rem;
+            font-weight: 900;
+            color: white;
+      }
+
+      .search-bar{
+            font-size: 1.2rem;
+            width: 300px!important;
+}
+
+      .search-icon{
+
+            position:absolute;
+            opacity: 0.6;
+            right: 310px;
+            bottom: 30px;
+            height: 20px;
+      }
+
+}
```

Comment:
Do correct the indentation of parenthesis in the media querries. It should be clear where the querry is ending.

Reply:
Fixed.